code/game/alternate_appearance.dm

```diff
@@ -29,12 +29,11 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 	appearance_key = key
 
 /datum/atom_hud/alternate_appearance/Destroy()
-	if(!QDELETED(src))
```

Comment:
I put this here because I was paranoid about infinite loops that turned out to not be a problem.